sqlflow/client.py

```diff
@@ -0,0 +1,69 @@
+import os
+import logging
+import grpc
+import google.protobuf.wrappers_pb2 as wrapper
+
+import sqlflow.proto.sqlflow_pb2
+import sqlflow.proto.sqlflow_pb2_grpc
+
+_LOGGER = logging.getLogger(__name__)
+
+class Client(object):
```

Comment:
(object) can be removed unless it supports python 2

Reply:
Done.